pkg/scheduler/core/spreadconstraint/group_clusters.go

```diff
@@ -137,10 +139,12 @@ func (info *GroupClustersInfo) generateClustersInfo(clustersScore framework.Clus
 		clusters = append(clusters, clusterScore.Cluster)
 	}
 
-	clustersReplicas := info.calAvailableReplicasFunc(clusters, rbSpec)
-	for i, clustersReplica := range clustersReplicas {
-		info.Clusters[i].AvailableReplicas = int64(clustersReplica.Replicas)
-		info.Clusters[i].AvailableReplicas += int64(rbSpec.AssignedReplicasForCluster(clustersReplica.Name))
+	if needAvailableReplicas {
```

Comment:
we can call the function here with `whetherNeedAvailableResource(rbSpec.Placement)`